g11n-ws/vip-manager-i18n/src/main/resources/application-s3.properties

```diff
@@ -0,0 +1,80 @@
+##
+#Copyright 2019 VMware, Inc.
+#SPDX-License-Identifier: EPL-2.0
+##
+
+# SPRING CONFIG (ConfigFileApplicationListener)
+spring.config.name= # config file name (default to 'application')
+spring.config.location= # location of config file
+
+# IDENTITY (ContextIdApplicationContextInitializer)
+spring.application.name=Singleton
+
+# logging
+logging.config=classpath:log4j2-spring.xml
+
+# EMBEDDED SERVER CONFIGURATION (ServerProperties)
+server.port=8090
+server.scheme=https
+server.https.key-store=classpath:vip.jks
+server.https.key-store-password=vmware2018
+server.https.key-password=vmware2018
+server.https.key-store-type=JKS
+server.https.key-alias=server
+server.http.port=8091
+server.trace.enable=false
+
+#collection source server
+source.cache.flag = false
+source.cache.server.url = https://localhost:8088
+
+#S3 store config
+s3.accessKey=###########
+s3.secretkey=##########
+s3.region=############
+s3.bucketName=##########
+
+
+#translation config
+#the follow item the Directory can't end of file separator
+translation.bundle.file.basepath =.
+translation.synch.git.flag = true
+
+#pseudo.enabled=false
+#pseudo.locale=latest
+pseudo.existSourceTag=#@
+pseudo.notExistSourceTag=@@
+
+#CSP authentication on/off default false
+csp.api.auth.enable=false
+##vmware internal csp authentication url
+csp.auth.url=####
+
+#authority config
+vipservice.authority.enable=false
+# the time unit is minutes
+vipservice.authority.session.expiretime=30
+#the time unit is hour
+vipservice.authority.token.expiretime=130
+
+#cross-domain configuration
+vipservice.cross.domain.enable=true
+vipservice.cross.domain.allowCredentials=true
+vipservice.cross.domain.alloworigin=*
+vipservice.cross.domain.allowmethods=GET, POST, PUT, DELETE, OPTIONS
+vipservice.cross.domain.allowheaders=csp-auth-token, Content-Type, x-xmp-ui, Authorization
+vipservice.cross.domain.maxage=3600
+
+#swagger2-ui switch
+swagger-ui.enable=true
+
+#microsoftstore mt config
+#mt.server=
+#mt.key=
+#mt.translatedMax=
+#mt.translatedCount=
```

Comment:
please put the original values back to these keys:
#mt.server=
#mt.translatedMax=
#mt.translatedCount=